halisaha_detay_2

Mr-Ztrk opened the pull request:
Halı saha detay güncel.

Mr-Ztrk committed:
halisaha_detay_2

mehmethayrullahozkul commented on src/pages/court-details/CourtDetails.tsx:
```diff
@@ -1,472 +1,904 @@
 
-const CourtDetails = (): JSX.Element => {
-  const params = useParams();
+import "../index.css";
```
Relative path ile import vermekten kaçınalım.

mehmethayrullahozkul commented on src/pages/court-details/CourtDetails.tsx:
```diff
@@ -1,472 +1,904 @@
-    },
+const SahanCepte: React.FC = () => {
+  const reviews = [
     {
```
`CourtDetails.tsx` sayfası neden `SahanCepte` olarak verilmiş?

mehmethayrullahozkul reviewed (changes requested):
eline sağlık ama 1-2 şey var

Mr-Ztrk committed:
Update CourtDetails.tsx

Mr-Ztrk committed:
Update CourtDetails.tsx

Mr-Ztrk committed:
Update CourtDetails.tsx

Commit:
dd

Commit:
hotfix: package conflicts resolved with merging on package.json
feat: Facility and Review models adapted to the backend views
feat: Facility details page fetched for available day slots
feat Reservation component added in simple form
